### PR TITLE
Fix sync to use correct bucket variable

### PR DESCRIPTION
**Pull request by keenahn**
Also added more details to README

### CODE REVIEW

**Comment by kewitz** [comment]
@keenahn, I've implemented the standard bucket name, I think it makes things easier. Check it out.

**Comment by keenahn** [comment]
This is awesome if it works. How do you auto-create the bucket though? I'll have to test this more later, but from just reading it it looks good.

File: deploy.js [hunk 1]
```diff
@@ -21,27 +18,31 @@ const verifyDependency = (condition, msg) => {
...
 const [ branch, ] = process.argv.slice(2)
 
 if (branch) {
   verifyDependency(GCLOUD_PROJECT, 'Sorry, you need to set a GCLOUD_PROJECT environment variable. You can set that in your local ".env" file.')
-  verifyDependency(GCLOUD_BACKEND_BUCKET, 'Sorry, you need to set a GCLOUD_BACKEND_BUCKET environment variable. You can set that in your local ".env" file.')
   verifyDependency(shell.which('gcloud'), 'Sorry, you need Google Cloud SDK installed. Get it on https://cloud.google.com/sdk/gcloud/')
   verifyDependency(shell.which('gsutil'), 'Sorry, you need "gsutil" Google Cloud Storage CLI installed. Get it on https://cloud.google.com/storage/docs/gsutil')
-  verifyDependency(shell.test('-e', `${branch}.yaml`), `Sorry, I couldn't find "${branch}.yaml"`)
 
+  const bucket = `gs://${GCLOUD_PROJECT}-backend`
```
Comment (keenahn) [inline deploy.js]: So here you're using the `-backend` suffix, but does that work? In the code for `deploy setup` it just uses `GCLOUD_PROJECT` so which is it?

Comment (keenahn) [inline deploy.js]: @kewitz

**Comment by keenahn** [comment]
I made one change in the README, but looks good. Merging

**Pull request merged**